### ISSUE TITLE
Change Board: product actions

### ISSUE THREAD

**Issue by kfklein15** [CodeYourFuture/CYF-Staff-Tickets#111]
Our [Change Board](https://github.com/CodeYourFuture/Changes) has been finally created
The historical document is this
We will start using it to track small changes we could make by contracting some of our graduates.

**Comment by kfklein15** [comment]
To be defined:
1 How to offer/communicate these opportunities
2 How to interview/define which graduate gets it

**Comment by SallyMcGrath** [comment]
1. We can subscribe a slack channel to the change board and new tickets will get posted on slack like our various product channels or #cyf-syllabus-tech-feed 
2. I don't know the answer to this. When I've tried advertising jobs it's just been a HUGE timesuck and has never panned out.  The only time I've gotten people to build/deliver things successfully is by approaching developers I already know to contract directly. The massive timesuck of the Jumpstart idea for Linux laptops -- never got anyone to just get on and BUILD it, just hours/days wasted! 🤷

**Comment by kfklein15** [comment]
Thanks, Sally. I am worried we will use favouritism to choose the developers, which is the opposite of what we always preach. So I don't have any ideas yet but I will put this on my thinking list.

**Comment by gbencci** [comment]
My suggestion on devs hiring:
1. Prioritise the product that needs to be created and what's most important for the organisation
2. Create the smallest product that we can create and to last no more than 1-3 months to launch
3. Set a high skills bar entry for creating the first product version. This will allow the critical parts to be done at production level
4. Coordinate with Employment, Product and Regional Teams on bringing people forward and ensure we have people from all the regions available

**Comment by kfklein15** [comment]
We have the following projects on the Change board (they haven't been broken down yet).
![image](https://user-images.githubusercontent.com/80272258/221587693-aee18737-dc79-4146-9aa2-3efd8ca76d16.png)

Which one do you think it the highest priority one, @SallyMcGrath and @gbencci ?

**Comment by SallyMcGrath** [comment]
1. The github app which I have already started with Berkeli is the highest impact for Syllabus, Tech Ed, Trainees, Proms, and Vols. As shown on ticket, we've already kicked this off.
2. Deploying the website would be high impact for the org and is a small job, it just needs someone to actually _do_ it. Small resource cost, high impact.
3.  Jumpstart would be quite high impact, to prevent further Windows tragedies, but I've been totally unable to get anyone to _get on_ with this job, and have no more time to put into talking/writing up/interviewing/asking/begging etc myself 😂 😅 
4. Github action for lighthouse  is a tiny tiny thing, would raise standards. It's a small, good thing and would be a nice CV line for a grad. Small resource cost from us.
5. Attendance register would be great IF someone could build a functional one, but Products would need to adopt it. Moderate impact for student metrics, moderate impact on toil for Proms and vols, high resources from us atm.
6. Final Projects is a dependent ticket on website deployment so ignore for now
